mep/people/views.py

```diff
@@ -126,6 +137,17 @@ def get_queryset(self):
         # throw a 404 if a non-member is accessed via this route
         return super().get_queryset().exclude(account=None)
 
+    def get_uri(self):
+        return self.object.get_absolute_url()
```

Comment:
I like the use of object `get_absolute_url` here.  

It occurs to me that if we want a smarter default implementation, we should be able to generate the current url based on a named route and request args. But that seems like a later refactor we should do if it makes sense.  (Or maybe `object.get_absolute_url` should be the generic for a `DetailView`.)